addon/templates/components/sl-date-range-picker.hbs

```diff
@@ -18,6 +19,7 @@
         placeholder=endDatePlaceholder
         startDate=earliestEndDate
         value=endDateValue
+        format=format
```

Comment:
If this is going to be a property that we support and a user can pass to the component, which is fine, then please also add an entry to the documentation at /tests/dummy/app/templates/demos/sl-date-range-picker.hbs